core/build.gradle

```diff
@@ -229,4 +234,4 @@ class PublicSuffixesTask extends DefaultTask {
 }
 
 task publicSuffixes(type: PublicSuffixesTask)
-tasks.processResources.dependsOn publicSuffixes
+tasks.classes.dependsOn publicSuffixes
```

Comment:
Isn't `processResources` better?

Reply:
When it was `processResources` the CI failed and seemed like `classes` worked. I'm not proficient at this so please let me know what works 😬.
https://ci.appveyor.com/project/line/armeria/builds/36497460/job/d0ji8ok3dnj2e384#L966

Reply:
Ah, OK. It's perhaps because we now write to `src/main`. Never mind, then.